src/StarkEx.Crypto.SDK/Hashing/SpotTradingMessageHasher.cs

```diff
@@ -55,6 +55,29 @@ public BigInteger EncodeLimitOrderWithFees(EncodeLimitOrderWithFeesModel encodeL
         return pedersenHash.CreateHash(thirdInnerHash, fifthWeight);
     }
 
+    /// <inheritdoc />
+    public BigInteger EncodeLimitOrderWithoutFees(EncodeLimitOrderWithoutFeesModel encodeLimitOrderWithoutFeesModel)
```

Comment:
This is functionally equivalent to `DeprecatedHashLimitOrder`.

Reply:
Deleted the existing deprecated method in favor of this one